web/src/pages/Home/TestList.tsx

```diff
@@ -62,6 +65,37 @@ const TestList = () => {
           );
         }}
       />
+      <Table.Column
+        title="Actions"
+        key="actions"
+        align="right"
+        render={i => {
+          return (
+            <Dropdown
+              overlay={
+                <Menu>
+                  <Menu.Item
+                    onClick={async e => {
+                      e.domEvent.stopPropagation();
```

Comment:
What do you think about moving this to a function? like `onDelete`?

Reply:
done!